synapse-store-tool.cwl

```diff
@@ -49,3 +50,10 @@ outputs:
     type: File
     outputBinding:
       glob: stdout.txt
+
+  - id: file_id
```

Comment:
Add a comment here to explain the regex and expected output of the `synapse store` command.